BLUESPAWN-common/headers/common/wrappers.hpp

```diff
@@ -6,6 +6,16 @@
 #include <memory>
 #include <optional>
 #include <functional>
+#include <lm.h>
```

Comment:
These includes are unneeded. Since wrappers.hpp is included in so much of bluespawn, it's best to limit includes as much as possible

Reply:
Nevermind, they're used in the shared_ptr. They still need to go, but I can take care of that eventually